index.js

```diff
@@ -64,6 +64,7 @@ function flameGraph (opts) {
   var focusedFrame = null
   var hoverFrame = null
   var currentAnimation = null
+  var noTooltip = opts.noTooltip || false
```

Comment:
Can we use an approach like with colorHash, where a custom renderTooltip function is passed in? 

So, we can disable the default tooltip by passing in an empty function; and also, it means 3rd parties implementing their own tooltips like Flame have a way to access all the d3-fg private variables and states, without needing more PRs exposing additional args.

Reply:
yes, sounds like a good idea.